static/usage/datetime/buttons/customizing-buttons/javascript.md

```diff
@@ -0,0 +1,13 @@
+```html
+<ion-datetime>
+  <ion-buttons slot="buttons">
+    <ion-button color="danger" onclick="datetime.reset()">Reset</ion-button>
+    <ion-button color="primary" onclick="datetime.cancel()">Never mind</ion-button>
+    <ion-button color="primary" onclick="datetime.confirm()">All Set</ion-button>
+  </ion-buttons>
+</ion-datetime>
+
+<script>
+  const datetime = document.querySelector('ion-datetime');
```

Comment:
```suggestion
  var datetime = document.querySelector('ion-datetime');
```

Stackblitz limitation... if you click any of the buttons in your live example you will get an exception. For whatever reason they don't allow `const` or `let` at the document scope.